### ISSUE TITLE
ImportError: cannot import name 'triu' from 'scipy.linalg'

### ISSUE THREAD

**Issue by nici7** [iomega/ms2query#245]
Hello,

I just installed MS2Query as described which seemed to be OK. (I created the envorinment in miniconda3 on a Windows PC.)

But when I run `ms2query --help` , or the command to download the library, I have the following issue:
```
(ms2query) C:\>ms2query --help
Traceback (most recent call last):
  File "C:\Users\thomenu\AppData\Local\miniconda3\envs\ms2query\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\thomenu\AppData\Local\miniconda3\envs\ms2query\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\thomenu\AppData\Local\miniconda3\envs\ms2query\Scripts\ms2query.exe\__main__.py", line 4, in <module>
  File "C:\Users\thomenu\AppData\Local\miniconda3\envs\ms2query\lib\site-packages\ms2query\__init__.py", line 9, in <module>
    from .ms2library import MS2Library, create_library_object_from_one_dir
  File "C:\Users\thomenu\AppData\Local\miniconda3\envs\ms2query\lib\site-packages\ms2query\ms2library.py", line 5, in <module>
    from gensim.models import Word2Vec
  File "C:\Users\thomenu\AppData\Local\miniconda3\envs\ms2query\lib\site-packages\gensim\__init__.py", line 11, in <module>
    from gensim import parsing, corpora, matutils, interfaces, models, similarities, utils  # noqa:F401
  File "C:\Users\thomenu\AppData\Local\miniconda3\envs\ms2query\lib\site-packages\gensim\corpora\__init__.py", line 6, in <module>
    from .indexedcorpus import IndexedCorpus  # noqa:F401 must appear before the other classes
  File "C:\Users\thomenu\AppData\Local\miniconda3\envs\ms2query\lib\site-packages\gensim\corpora\indexedcorpus.py", line 14, in <module>
    from gensim import interfaces, utils
  File "C:\Users\thomenu\AppData\Local\miniconda3\envs\ms2query\lib\site-packages\gensim\interfaces.py", line 19, in <module>
    from gensim import utils, matutils
  File "C:\Users\thomenu\AppData\Local\miniconda3\envs\ms2query\lib\site-packages\gensim\matutils.py", line 20, in <module>
    from scipy.linalg import get_blas_funcs, triu
ImportError: cannot import name 'triu' from 'scipy.linalg' (C:\Users\thomenu\AppData\Local\miniconda3\envs\ms2query\lib\site-packages\scipy\linalg\__init__.py)
```

How can this be fixed?
Thanks!

**Comment by florian-huber** [comment]
There is a known incompatibility between certain gensim versions and newer scipy version.
Usually this can be fixed by either updating gensim, or downgrading scipy (to <1.13 I believe).

**Comment by florian-huber** [comment]
Unfortunately, we cannot work on this for the coming weeks. I expect that mid- or end of august @niekdejonge or I will get to it.

**Comment by nici7** [comment]
Thanks, that helps already! I downgraded the scipy version in the environment. Now `ms2query --help` works as it should. 

When testing with "dummy_spectra.mgf" though, I run into the same issue as described here: https://github.com/iomega/ms2query/issues/244
```
AttributeError: module 'matchms.filtering' has no attribute 'add_losses'
```

**Comment by wjcranda** [comment]
@nici7 The following package version should allow you to successfully run MS2query v1.5.0 with the latest models and libraries on Zenodo. Scipy must be downgraded as @florian-huber stated and your next error encountered results from matchms.filtering 'add_loses' no longer being a feature as of the latest update (see matchms 0.27.0 change log). So matchms must be downgraded.  gensim version must be a compatible version with downgraded scipy. 

numpy: 1.24.0
matchms: 0.26.4
ms2query: 1.5.0
gensim: 4.3.2
scipy: 1.10.1

**Comment by niekdejonge** [comment]
Thanks @wjcranda for helping out! And yes you are correct. I am back from holidays now, so I will fix it within MS2Query as well, version 1.5.1 should be fully fixed again (not yet released). Sorry for the inconvenience.

**Comment by niekdejonge** [comment]
The add_losses issue should be fixed for ms2query = 1.5.1. Feel free to reopen the issue if it is not resolved for you yet @nici7
